apcd-cms/src/apps/submission_form/templates/submission_form/submission_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+<div>
+    <h3>Submission Form</h3>
+    <form action="" method="POST">
```

Comment:
**Idea.** We may be able to just use `<form method="POST">` ([ref 1](https://stackoverflow.com/a/13520198/11817077), [ref 2](https://stackoverflow.com/a/16800092/11817077)). I'll test.